### PR TITLE
fix invalid worker kind issue

### PR DESCRIPTION
**Pull request by hougangliu**
studyJob should go to 'Failed' status when worker kind is invalid

fixes: #285

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/287)

### CODE REVIEW

**Comment by johnugeorge** [comment]
Tests are failing because of #286 

Also, please hold till #283 is merged

File: pkg/controller/studyjob/const.go [hunk 1]
```diff
@@ -15,3 +15,5 @@ const (
 	DefaultJobWorker = "Job"
 	TFJobWorker = "TFJob"
 )
+
+var ValidWorkerKindList = [...]string{DefaultJobWorker, TFJobWorker}
```
[the remark this answers is not in the document]
Comment (hougangliu) [reply pkg/controller/studyjob/const.go]: done! thanks

**Pull request merged**